src/property-filter/controller.ts

```diff
@@ -66,7 +66,12 @@ export const getAllowedOperators = (
   const { operators, defaultOperator } = property;
   const operatorOrder = ['=', '!=', ':', '!:', '>=', '<=', '<', '>'] as const;
   const operatorSet: { [key: string]: true } = { [defaultOperator ?? '=']: true };
-  operators?.forEach(op => (operatorSet[op] = true));
+  // TODO: remove unknown type annotation once extended operators are supported.
+  operators?.forEach((op: unknown) => {
+    if (typeof op === 'string') {
```

Comment:
This temporarily fixes the following failure: https://github.com/cloudscape-design/collection-hooks/pull/21.

Once extended operator types are supported by the property filter, the `unknown` annotation can be removed and the code changed to:

```
operators.forEach(op => {
  if (typeof op === 'string') {
    operatorsSet[op] = true
  } else {
    operatorsSet[op.value] = true
  }
})
```